perllib/FixMyStreet/App/Controller/Admin/Users.pm

```diff
@@ -209,8 +209,12 @@ sub edit : Path : Args(1) {
     }
 
     $c->forward('/admin/fetch_all_bodies');
-    $c->forward('/admin/fetch_body_areas', [ $user->from_body ]) if $user->from_body;
-    $c->forward('fetch_body_roles', [ $user->from_body ]) if $user->from_body;
+    if ($user->from_body) {
+        unless ($c->cobrand->call_hook(admin_fetch_inspector_areas => $user->from_body)) {
```

Comment:
This is okay, but I think would be better if this hook wasn't here (and below) but inside `admin/fetch_body_areas`, and that way the hook wouldn't have to duplicate the name sorting or setting of fetched_areas_body_id.